### PR TITLE
update Permissions API

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

add {{AvailableInWorkers}} to `Permissions` interface and `PermissionStatus` interface and their related pages

add missing `local-fonts` permission descriptor

update `Permissions.query()` exception section - style change from html table to markdown list and add unwritten `InvalidStateError`

### Additional details

reference to https://w3c.github.io/permissions/

### CODE REVIEW

**Comment by skyclouds2001** [comment]
also, when I'm checking issues, it seems that https://github.com/mdn/content/issues/27341 has alreadly fixed

File: files/en-us/web/api/permissions/query/index.md [hunk 1]
```diff
@@ -39,25 +39,10 @@ A {{jsxref("Promise")}} that resolves to a {{domxref("PermissionStatus")}} objec
...
-        (e.g. <code>midi</code>, or <code>push</code> with
-        <code>userVisibleOnly</code>).
-      </td>
-    </tr>
-  </tbody>
-</table>
+- `InvalidStateError` {{domxref("DOMException")}}
+  - : Thrown if `query()` method is invoked in the browsing context and its associated document is not fully active.
+- {{jsxref("TypeError")}}
+  - : Thrown if retrieving the `PermissionDescriptor` information failed in some way, or the permission doesn't exist or is currently unsupported (e.g. `midi`, or `push` with `userVisibleOnly`).
```
Comment (wbamberg) [inline files/en-us/web/api/permissions/query/index.md:45]: >  (e.g. `midi`, or `push` with `userVisibleOnly`)

What does this mean? AFAIK `midi` exists, and whether it is supported depends on the browser. And I don't know what "`push` with `userVisibleOnly`".

I would be OK just to omit this, unless we can explain what it means.

Comment (skyclouds2001) [reply files/en-us/web/api/permissions/query/index.md:45]: the Push API extended the `PushPermissionDescriptor` of `PermissionDescriptor` to allow passing an extra param `userVisibleOnly`, see https://w3c.github.io/push-api/#permission

**Review by wbamberg — APPROVED** [review]
👍 thank you @skyclouds2001 !

**Pull request merged**